$NetBSD$

use dot macros for some because cpp did not recognize backslash sequences

--- general/X.man.orig	2003-10-27 16:54:26.000000000 -0800
+++ general/X.man	2003-10-27 16:59:11.000000000 -0800
@@ -505,7 +505,7 @@
 If the name is not found, the color is looked up in the
 X server's database.
 The text form of this database is commonly stored in the file
-\fI__projectroot__/lib/X11/rgb.txt\fP.
+.IR __projectroot__/lib/X11/rgb.txt .
 .PP
 A numerical color specification
 consists of a color space name and a set of values in the following syntax:
@@ -1016,12 +1016,14 @@
 .B XLOCALEDIR
 This must point to a directory containing the locale.alias file and
 Compose and XLC_LOCALE file hierarchies for all locales. The default value
-is \fI__projectroot__/lib/X11/locale\fP.
+is
+.IR __projectroot__/lib/X11/locale .
 .TP
 .SM
 .B XENVIRONMENT
 This must point to a file containing X resources. The default is
-\fI$HOME/.Xdefaults-<hostname>\fP. Unlike \fI__projectroot__/lib/X11/Xresources\fP,
+\fI$HOME/.Xdefaults-<hostname>\fP. Unlike
+.IR __projectroot__/lib/X11/Xresources ,
 it is consulted each time an X application starts.
 .TP
 .SM
@@ -1094,17 +1096,19 @@
 .SM
 .B XKEYSYMDB
 This must point to a file containing nonstandard keysym definitions.
-The default value is \fI__projectroot__/lib/X11/XKeysymDB\fP.
+The default value is
+.IR __projectroot__/lib/X11/XKeysymDB .
 .TP
 .SM
 .B XCMSDB
 This must point to a color name database file. The default value is
-\fI__projectroot__/lib/X11/Xcms.txt\fP.
+.IR __projectroot__/lib/X11/Xcms.txt .
 .TP
 .SM
 .B XFT_CONFIG
 This must point to a configuration file for the Xft library. The default
-value is \fI__projectroot__/lib/X11/XftConfig\fP.
+value is
+.IR __projectroot__/lib/X11/XftConfig .
 .TP
 .SM
 .B RESOURCE_NAME
@@ -1186,7 +1190,8 @@
 The default error handler in \fIXlib\fP (also used by many toolkits) uses
 standard resources to construct diagnostic messages when errors occur.  The
 defaults for these messages are usually stored in
-\fI__projectroot__/lib/X11/XErrorDB\fP.  If this file is not present,
+.IR __projectroot__/lib/X11/XErrorDB .
+If this file is not present,
 error messages will be rather terse and cryptic.
 .PP
 When the X Toolkit Intrinsics encounter errors converting resource strings to
